rasa_core/policies/embedding_policy.py

```diff
@@ -0,0 +1,1576 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import io
+import os
+import logging
+import warnings
+import typing
+from tqdm import tqdm
+
+from typing import \
+    Any, List, Optional, Text
+
+import numpy as np
+import copy
+from rasa_core.policies import Policy
+from rasa_core.featurizers import \
+    TrackerFeaturizer, FullDialogueTrackerFeaturizer, \
+    LabelTokenizerSingleStateFeaturizer
+from rasa_core import utils
+
+if typing.TYPE_CHECKING:
+    from rasa_core.domain import Domain
+    from rasa_core.trackers import DialogueStateTracker
+
+try:
+    import cPickle as pickle
+except ImportError:
+    import pickle
+
+try:
+    import tensorflow as tf
+except ImportError:
+    tf = None
+
+logger = logging.getLogger(__name__)
+
+
+class EmbeddingPolicy(Policy):
+    SUPPORTS_ONLINE_TRAINING = True
+
```

Comment:
have we tried out online training? not a priority rn but interested what form it should take

Reply:
what do you mean? this is our interactive learning, I checked that it technically works sometime ago, might need to do it again before merging